ethereum-bridged-carbon/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "ethereum-bridged-carbon",
+  "license": "UNLICENSED",
```

Comment:
This should be "MIT" license since that's what we've licensed this whole repo under